src/Uno.UWP/ApplicationModel/Contacts/ContactBatch.cs

```diff
@@ -0,0 +1,24 @@
+using System;
+using System.Collections.Generic;
+using System.Runtime.Serialization;
+using System.Threading.Tasks;
+using Windows.Foundation;
+
+namespace Windows.ApplicationModel.Contacts
+{
+
+	public partial class ContactBatch
+	{
+
+		public IReadOnlyList<Contact> Contacts { get; }
+
+#if __ANDROID__
```

Comment:
There is no need to put this Android specific.

Reply:
It is used in Android implementation, and I know nothing about other implementations.
But ok.